src/transform/WidenImage.css

```diff
@@ -68,4 +68,9 @@ li.gallerybox div.thumb > div {
 }
 div.gallerytext > p {
   margin-top: 0px !important;
+}
+
+/* Prevent image placeholder to be wider than the viewport */
+.gallerybox .pagelib_lazy_load_placeholder {
+  width: 100%!important;
```

Comment:
Would you add a space between the `%` and the `!`?
Have you considered using `max-width` instead of `width`?

Reply:
> Would you add a space between the `%` and the `!`?

Done.
> Have you considered using `max-width` instead of `width`?

It's already applied https://github.com/wikimedia/wikimedia-page-library/blob/master/src/transform/LazyLoadTransform.css#L38

Reply:
Good catch! Thanks!